pydis_site/apps/main/models/repo_data.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from django.utils import timezone
+
+
+class RepoData(models.Model):
```

Comment:
wouldn't `Repository` or `RepositoryMetadata` a more descriptive name for this?

Reply:
sure, RepositoryMetadata is pretty good I guess.